views/index.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Chitter</h1>
```

Comment:
If you wanted this to be displayed on every page an easy way would be to extract it into a layout erb and place the h1 in the header